src/server/rdb_test.cc

```diff
@@ -655,4 +655,14 @@ TEST_F(RdbTest, SnapshotTooBig) {
   ASSERT_THAT(resp, ErrArg("Out of memory"));
 }
 
+TEST_F(RdbTest, LoadStream2) {
+  auto ec = LoadRdb("RDB_TYPE_STREAM_LISTPACKS_2.rdb");
```

Comment:
please expand the tests to actually check that the streams are present and they have items.